packages/tldraw/src/lib/ui.css

```diff
@@ -1039,10 +1039,14 @@ tldraw? probably.
 	display: none;
 }
 
-.tlui-style-panel[data-ismobile='true'] .tlui-style-panel__section .tlui-row.tlui-toolbar {
+.tlui-style-panel__section .tlui-toolbar:has(.tlui-toolbar) {
```

Comment:
this "thing contains itself" selector is a little confusing and could maybe use a comment